tests/ragger/test_appname_cmd.py

```diff
@@ -0,0 +1,2 @@
+def test_app_name(cmd):
+    assert cmd.get_app_name() == "Boilerplate"
```

Comment:
No way to grab APPNAME from the Makefile ?

Reply:
I don't see how to do it with unecessary burden otherwise. For instance there could be a `pytest` target in the `Makefile` which would trigger the test with env variables (containing the app name, version or whatenot), but that would mean the preferred test entrypoint would be `make`, not `pytest`, and manipulating the test directly would need to explicitely set environment variables, which with time could be numerous.

TLD;DR: Don't know if it is worth it